content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -72,213 +114,233 @@ Once the :guilabel:`Lot/Serial Number` is generated, click the blank field next
 :guilabel:`Product` to reveal a drop-down menu. From this menu, select the product to which this new
 number will be assigned.
 
-This form also provides the option to adjust the :guilabel:`Quantity`, to assign a unique
-:guilabel:`Internal Reference` number (for traceability purposes), and to assign this specific
-lot/serial number configuration to a specific website in the :guilabel:`Website` field (if working
-in a multi-website environment).
-
-A detailed description of this specific lot/serial number can also be added in the
+This form also provides the option to adjust the :guilabel:`On Hand Quantity`, to assign a unique
+:guilabel:`Internal Reference` number (for additional traceability), and to assign this specific
+lot/serial number configuration to a specific company in the :guilabel:`Company` field. A detailed
+description of this specific lot/serial number can also be added in the
 :guilabel:`Description` tab below.
 
 When all desired configurations are complete, click the :guilabel:`Save` button to save all changes.
 
-.. image:: serial_numbers/serial-numbers-new-serial-number.png
+.. image:: serial_numbers/new-serial-number.png
```

Comment:
Can retake this screenshot in HD or only crop the necessary portion?
It's a bit too wide / zoomed out